### PR TITLE
Add service for Issuu (www.issuu.com) Iframe embed

### PR DESCRIPTION
**Pull request by achterbahn**
Added Service for Issuu PDF Viewer when used with iframe embed

Usage:

1. Add this
`(tarteaucitron.job = tarteaucitron.job || []).push('issuu');`

2. Replacement code 
`<div class="issuu_player" issuuID="your_issuu_id" width="width" height="height"></div>`

3. Service marker (delete if installed)
`<iframe style="width:400px; height:300px;" src="//e.issuu.com/embed.html#your_issue_id" frameborder="0" allowfullscreen></iframe>`

### CODE REVIEW

File: tarteaucitron.services.js [hunk 1]
```diff
@@ -1848,3 +1848,48 @@ tarteaucitron.services.facebookpixel = {
     }
 };
 
+//Issuu
+tarteaucitron.services.issuu = {
+    "key": "issuu",
+    "type": "other",
```
Comment (AmauriC) [inline tarteaucitron.services.js]: I don't know this service. That's better to avoid "other". Ads is ok?

**Comment by AmauriC** [comment]
Thanks :+1:

Can you send me a screenshot of the service in action for the doc?
> https://opt-out.ferank.eu/img/services/googlepartners.png

File: tarteaucitron.services.js (hunk 1 above)
Comment (achterbahn) [reply tarteaucitron.services.js]: Issuu is a service to display PDFs embeded into your website. 
So, "ads" is not the right place to put it, I think.

**Comment by achterbahn** [comment]
issuu

File: tarteaucitron.services.js (hunk 1 above)
Comment (AmauriC) [reply tarteaucitron.services.js]: Ok, set the type to video like calameo and prezi and I will change this later to "media".

**Pull request merged**